src/output.c

```diff
@@ -176,9 +176,20 @@ new_output_notify(struct wl_listener *listener, void *data)
 	wlr_scene_node_raise_to_top(&output->layer_popup_tree->node);
 
 	if (rc.adaptive_sync) {
-		wlr_log(WLR_INFO, "enable adaptive sync on %s",
-			wlr_output->name);
-		wlr_output_enable_adaptive_sync(wlr_output, true);
+		wlr_log(WLR_INFO, "enable adaptive sync on %s", wlr_output->name);
+		struct wlr_output_state pending = { 0 };
+		wlr_output_state_set_adaptive_sync_enabled(&pending, true);
+		if (!wlr_output_test_state(wlr_output, &pending)) {
+			wlr_log(WLR_ERROR, "adaptive sync failed, ignoring");
+			wlr_output_state_set_adaptive_sync_enabled(&pending, false);
+		} else {
+			wlr_log(WLR_INFO, "adaptive sync test passed");
+		}
+		if (!wlr_output_commit_state(wlr_output, &pending)) {
+			wlr_log(WLR_ERROR, "failed to commit output %s, "
+				"destroying", wlr_output->name);
+			output_destroy_notify(&output->destroy, NULL);
```

Comment:
Shouldn't we just disable `rc.adaptive_sync` here and move on? On my test system (not supporting adaptive sync) this PR (with enabled `adaptiveSync` in config) results in being unable to start labwc at all because `wlr_output_test_state()` passes for both outputs but committing fails so there is no output available (and then additionally segfaults, likely because of no output being available).

```
00:00:01.494 [INFO] [../src/output.c:186] adaptive sync test passed
00:00:01.494 [DEBUG] [backend/drm/drm.c:520] connector VGA-1: Failed to enable adaptive sync: connector doesn't support VRR
00:00:01.494 [ERROR] [../src/output.c:189] failed to commit output VGA-1, destroying
```

<details><summary>debug log</summary>

```
00:00:00.000 [INFO] [../src/main.c:124] using config dir (/home/benutzer/.config/labwc)
00:00:00.002 [INFO] [../src/config/session.c:63] read environment file /home/benutzer/.config/labwc/environment
00:00:00.003 [INFO] [../src/config/rcxml.c:755] read config file /home/benutzer/.config/labwc/rc.xml
00:00:00.005 [DEBUG] [../src/config/rcxml.c:637] Loaded 28 merged mousebinds
00:00:00.006 [INFO] [../src/config/rcxml.c:117] create mousebind for Root
00:00:00.006 [DEBUG] [../src/config/rcxml.c:662] Replaced 1 mousebinds
00:00:00.007 [DEBUG] [../src/main.c:138] LABWC_PID=2071284
00:00:00.009 [INFO] [libseat] [libseat/backend/seatd.c:64] Could not connect to socket /run/seatd.sock: Datei oder Verzeichnis nicht gefunden
00:00:00.010 [INFO] [libseat] [libseat/libseat.c:76] Backend 'seatd' failed to open seat, skipping
00:00:00.138 [INFO] [libseat] [libseat/libseat.c:73] Seat opened with backend 'logind'
00:00:00.138 [INFO] [backend/session/session.c:109] Successfully loaded libseat session
00:00:00.152 [INFO] [backend/backend.c:220] Found 1 GPUs
00:00:00.153 [INFO] [backend/drm/backend.c:204] Initializing DRM backend for /dev/dri/card0 (i915)
00:00:00.153 [DEBUG] [backend/drm/drm.c:84] Atomic modesetting unsupported, using legacy DRM interface
00:00:00.153 [DEBUG] [backend/drm/drm.c:100] ADDFB2 modifiers supported
00:00:00.153 [INFO] [backend/drm/drm.c:253] Found 2 DRM CRTCs
00:00:00.154 [INFO] [backend/drm/drm.c:180] Found 4 DRM planes
00:00:00.154 [INFO] [util/env.c:25] Loading WLR_RENDERER option: gles2
00:00:00.159 [INFO] [render/egl.c:201] Supported EGL client extensions: EGL_EXT_device_base EGL_EXT_device_enumeration EGL_EXT_device_query EGL_EXT_platform_base EGL_KHR_client_get_all_proc_addresses EGL_EXT_client_extensions EGL_KHR_debug EGL_EXT_platform_device EGL_EXT_platform_wayland EGL_KHR_platform_wayland EGL_EXT_platform_x11 EGL_KHR_platform_x11 EGL_EXT_platform_xcb EGL_MESA_platform_gbm EGL_KHR_platform_gbm EGL_MESA_platform_surfaceless
00:00:00.167 [DEBUG] [render/egl.c:469] Using EGL device /dev/dri/card0
00:00:00.364 [INFO] [render/egl.c:347] Using EGL 1.4
00:00:00.364 [INFO] [render/egl.c:348] Supported EGL display extensions: EGL_ANDROID_blob_cache EGL_EXT_image_dma_buf_import EGL_EXT_image_dma_buf_import_modifiers EGL_KHR_cl_event2 EGL_KHR_config_attribs EGL_KHR_context_flush_control EGL_KHR_create_context EGL_KHR_create_context_no_error EGL_KHR_fence_sync EGL_KHR_get_all_proc_addresses EGL_KHR_gl_renderbuffer_image EGL_KHR_gl_texture_2D_image EGL_KHR_gl_texture_3D_image EGL_KHR_gl_texture_cubemap_image EGL_KHR_image_base EGL_KHR_no_config_context EGL_KHR_reusable_sync EGL_KHR_surfaceless_context EGL_EXT_pixel_format_float EGL_KHR_wait_sync EGL_MESA_configless_context EGL_MESA_drm_image EGL_MESA_image_dma_buf_export EGL_MESA_query_driver EGL_WL_bind_wayland_display 
00:00:00.364 [INFO] [render/egl.c:350] Supported EGL device extensions: EGL_EXT_device_drm EGL_EXT_device_drm_render_node
00:00:00.364 [INFO] [render/egl.c:352] EGL vendor: Mesa Project
00:00:00.364 [DEBUG] [render/egl.c:121] Supported DMA-BUF formats:
00:00:00.364 [DEBUG] [render/egl.c:165]   AR30 (0x30335241)
00:00:00.364 [DEBUG] [render/egl.c:104]     INVALID (0x00FFFFFFFFFFFFFF): ✓ texture  ✓ render
00:00:00.364 [DEBUG] [render/egl.c:104]     LINEAR (0x0000000000000000): ✓ texture  ✓ render
00:00:00.364 [DEBUG] [render/egl.c:165]   AR24 (0x34325241)
00:00:00.364 [DEBUG] [render/egl.c:104]     INVALID (0x00FFFFFFFFFFFFFF): ✓ texture  ✓ render
00:00:00.364 [DEBUG] [render/egl.c:104]     LINEAR (0x0000000000000000): ✓ texture  ✓ render
00:00:00.365 [DEBUG] [render/egl.c:165]   AB24 (0x34324241)
00:00:00.365 [DEBUG] [render/egl.c:104]     INVALID (0x00FFFFFFFFFFFFFF): ✓ texture  ✓ render
00:00:00.365 [DEBUG] [render/egl.c:104]     LINEAR (0x0000000000000000): ✓ texture  ✓ render
00:00:00.365 [DEBUG] [render/egl.c:165]   XR24 (0x34325258)
00:00:00.365 [DEBUG] [render/egl.c:104]     INVALID (0x00FFFFFFFFFFFFFF): ✓ texture  ✓ render
00:00:00.365 [DEBUG] [render/egl.c:104]     LINEAR (0x0000000000000000): ✓ texture  ✓ render
00:00:00.365 [DEBUG] [render/egl.c:165]   XB24 (0x34324258)
00:00:00.365 [DEBUG] [render/egl.c:104]     INVALID (0x00FFFFFFFFFFFFFF): ✓ texture  ✓ render
00:00:00.365 [DEBUG] [render/egl.c:104]     LINEAR (0x0000000000000000): ✓ texture  ✓ render
00:00:00.365 [DEBUG] [render/egl.c:165]   AR15 (0x35315241)
00:00:00.365 [DEBUG] [render/egl.c:104]     INVALID (0x00FFFFFFFFFFFFFF): ✓ texture  ✓ render
00:00:00.365 [DEBUG] [render/egl.c:104]     LINEAR (0x0000000000000000): ✓ texture  ✓ render
00:00:00.365 [DEBUG] [render/egl.c:165]   RG16 (0x36314752)
00:00:00.365 [DEBUG] [render/egl.c:104]     INVALID (0x00FFFFFFFFFFFFFF): ✓ texture  ✓ render
00:00:00.365 [DEBUG] [render/egl.c:104]     LINEAR (0x0000000000000000): ✓ texture  ✓ render
00:00:00.365 [DEBUG] [render/egl.c:165]   AYUV (0x56555941)
00:00:00.365 [DEBUG] [render/egl.c:104]     INVALID (0x00FFFFFFFFFFFFFF): ✓ texture  ✓ render
00:00:00.365 [DEBUG] [render/egl.c:104]     LINEAR (0x0000000000000000): ✓ texture  ✓ render
00:00:00.365 [DEBUG] [render/egl.c:165]   XYUV (0x56555958)
00:00:00.365 [DEBUG] [render/egl.c:104]     INVALID (0x00FFFFFFFFFFFFFF): ✓ texture  ✓ render
00:00:00.365 [DEBUG] [render/egl.c:104]     LINEAR (0x0000000000000000): ✓ texture  ✓ render
00:00:00.366 [DEBUG] [render/egl.c:165]   YUYV (0x56595559)
00:00:00.366 [DEBUG] [render/egl.c:104]     INVALID (0x00FFFFFFFFFFFFFF): ✓ texture  ✓ render
00:00:00.366 [DEBUG] [render/egl.c:104]     LINEAR (0x0000000000000000): ✓ texture  ✓ render
00:00:00.366 [DEBUG] [render/egl.c:165]   UYVY (0x59565955)
00:00:00.366 [DEBUG] [render/egl.c:104]     INVALID (0x00FFFFFFFFFFFFFF): ✓ texture  ✓ render
00:00:00.366 [DEBUG] [render/egl.c:104]     LINEAR (0x0000000000000000): ✓ texture  ✓ render
00:00:00.366 [DEBUG] [render/egl.c:185] EGL DMA-BUF format modifiers unsupported
00:00:00.375 [DEBUG] [render/egl.c:518] Using EGL_PLATFORM_DEVICE_EXT
00:00:00.387 [INFO] [render/gles2/renderer.c:718] Creating GLES2 renderer
00:00:00.387 [INFO] [render/gles2/renderer.c:719] Using OpenGL ES 2.0 Mesa 22.2.0
00:00:00.387 [INFO] [render/gles2/renderer.c:720] GL vendor: Mesa Project
00:00:00.387 [INFO] [render/gles2/renderer.c:721] GL renderer: i915 (chipset: Pineview M)
00:00:00.387 [INFO] [render/gles2/renderer.c:722] Supported GLES2 extensions: GL_EXT_blend_minmax GL_EXT_multi_draw_arrays GL_EXT_texture_filter_anisotropic GL_EXT_texture_compression_s3tc GL_EXT_texture_compression_dxt1 GL_EXT_texture_format_BGRA8888 GL_OES_compressed_ETC1_RGB8_texture GL_OES_depth24 GL_OES_element_index_uint GL_OES_fbo_render_mipmap GL_OES_mapbuffer GL_OES_rgb8_rgba8 GL_OES_stencil8 GL_OES_texture_3D GL_OES_texture_npot GL_OES_vertex_half_float GL_EXT_draw_instanced GL_OES_EGL_image GL_OES_depth_texture GL_OES_packed_depth_stencil GL_EXT_texture_type_2_10_10_10_REV GL_OES_get_program_binary GL_APPLE_texture_max_level GL_EXT_discard_framebuffer GL_EXT_read_format_bgra GL_NV_pack_subimage GL_EXT_frag_depth GL_NV_fbo_color_attachments GL_OES_EGL_image_external GL_OES_EGL_sync GL_OES_vertex_array_object GL_ANGLE_pack_reverse_row_order GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_EXT_unpack_subimage GL_NV_draw_buffers GL_NV_read_buffer GL_NV_read_depth GL_NV_read_depth_stencil GL_NV_read_stencil GL_EXT_draw_buffers GL_EXT_map_buffer_range GL_KHR_debug GL_NV_pixel_buffer_object GL_OES_required_internalformat GL_OES_surfaceless_context GL_EXT_separate_shader_objects GL_EXT_compressed_ETC1_RGB8_sub_texture GL_EXT_draw_elements_base_vertex GL_EXT_texture_border_clamp GL_KHR_context_flush_control GL_OES_draw_elements_base_vertex GL_OES_texture_border_clamp GL_KHR_no_error GL_EXT_texture_compression_s3tc_srgb GL_KHR_parallel_shader_compile GL_MESA_bgra 
00:00:00.437 [DEBUG] [GLES2] FS shader: 3 inst, 0 tex, 1 tex_indirect, 0 temps, 1 const
00:00:00.448 [DEBUG] [GLES2] FS shader: 6 inst, 1 tex, 1 tex_indirect, 1 temps, 1 const
00:00:00.461 [DEBUG] [GLES2] FS shader: 9 inst, 1 tex, 1 tex_indirect, 1 temps, 2 const
00:00:00.472 [DEBUG] [GLES2] FS shader: 6 inst, 1 tex, 1 tex_indirect, 1 temps, 1 const
00:00:00.475 [DEBUG] [render/allocator/allocator.c:106] Trying to create gbm allocator
00:00:00.482 [DEBUG] [render/allocator/gbm.c:210] Created GBM allocator with backend drm
00:00:00.482 [DEBUG] [render/allocator/gbm.c:213] Using DRM node /dev/dri/card0
00:00:00.483 [DEBUG] [types/wlr_idle.c:246] idle manager created
00:00:00.594 [DEBUG] [xcursor/wlr_xcursor.c:243] Loaded cursor theme 'default' at size 24 (124 available cursors)
00:00:00.595 [DEBUG] [types/wlr_drm_lease_v1.c:638] Creating wlr_drm_lease_device_v1 for /dev/dri/card0
00:00:00.596 [DEBUG] [../src/server.c:424] xwayland is running on display :0
00:00:00.597 [DEBUG] [backend/libinput/backend.c:87] Starting libinput backend
00:00:01.041 [DEBUG] [backend/libinput/events.c:70] Adding Power Button [0:1]
00:00:01.065 [DEBUG] [backend/libinput/events.c:70] Adding Video Bus [0:6]
00:00:01.098 [DEBUG] [backend/libinput/events.c:70] Adding Power Button [0:1]
00:00:01.131 [DEBUG] [backend/libinput/events.c:70] Adding Lid Switch [0:5]
00:00:01.134 [INFO] [../src/seat.c:284] unsupported input device
00:00:01.137 [DEBUG] [backend/libinput/events.c:70] Adding Sleep Button [0:3]
00:00:01.175 [DEBUG] [backend/libinput/events.c:70] Adding Logitech Wireless Keyboard PID:4023 [1133:16419]
00:00:01.211 [INFO] [../src/seat.c:84] tap unavailable
00:00:01.214 [INFO] [../src/seat.c:94] natural scroll not configured
00:00:01.217 [INFO] [../src/seat.c:103] left-handed mode not configured
00:00:01.220 [INFO] [../src/seat.c:111] pointer acceleration unavailable
00:00:01.223 [INFO] [../src/seat.c:126] middle emulation not configured
00:00:01.224 [INFO] [../src/seat.c:135] dwt not configured
00:00:01.225 [INFO] [../src/seat.c:170] map pointer to output (null)
00:00:01.226 [DEBUG] [backend/libinput/events.c:70] Adding Logitech Wireless Mouse [1133:16468]
00:00:01.227 [INFO] [../src/seat.c:84] tap unavailable
00:00:01.228 [INFO] [../src/seat.c:94] natural scroll not configured
00:00:01.229 [INFO] [../src/seat.c:103] left-handed mode not configured
00:00:01.230 [INFO] [../src/seat.c:113] pointer acceleration configured
00:00:01.231 [INFO] [../src/seat.c:128] middle emulation configured
00:00:01.231 [INFO] [../src/seat.c:135] dwt not configured
00:00:01.232 [INFO] [../src/seat.c:170] map pointer to output (null)
00:00:01.234 [DEBUG] [backend/libinput/events.c:70] Adding 1.3M WebCam: 1.3M WebCam [1026:38501]
00:00:01.268 [DEBUG] [backend/libinput/events.c:70] Adding AT Translated Set 2 keyboard [1:1]
00:00:01.305 [DEBUG] [backend/libinput/events.c:70] Adding SynPS/2 Synaptics TouchPad [2:7]
00:00:01.305 [INFO] [../src/seat.c:86] tap configured
00:00:01.306 [INFO] [../src/seat.c:94] natural scroll not configured
00:00:01.307 [INFO] [../src/seat.c:103] left-handed mode not configured
00:00:01.307 [INFO] [../src/seat.c:113] pointer acceleration configured
00:00:01.308 [INFO] [../src/seat.c:126] middle emulation not configured
00:00:01.309 [INFO] [../src/seat.c:137] dwt configured
00:00:01.310 [INFO] [../src/seat.c:170] map pointer to output (null)
00:00:01.310 [DEBUG] [backend/libinput/events.c:70] Adding Acer WMI hotkeys [0:0]
00:00:01.332 [DEBUG] [backend/libinput/backend.c:128] libinput successfully initialized
00:00:01.332 [INFO] [backend/drm/drm.c:1325] Scanning DRM connectors on /dev/dri/card0
00:00:01.333 [INFO] [backend/drm/drm.c:1376] Found connector 'LVDS-1'
00:00:01.333 [INFO] [backend/drm/drm.c:1401] 'LVDS-1' connected
00:00:01.333 [DEBUG] [backend/drm/drm.c:1207] Current CRTC: 44
00:00:01.333 [INFO] [util/env.c:9] Loading WLR_NO_HARDWARE_CURSORS option: 1
00:00:01.333 [DEBUG] [types/output/output.c:382] WLR_NO_HARDWARE_CURSORS set, forcing software cursors
00:00:01.333 [INFO] [backend/drm/drm.c:1217] Physical size: 220x130
00:00:01.333 [INFO] [backend/drm/drm.c:1274] Detected modes:
00:00:01.333 [INFO] [backend/drm/drm.c:1302]   1024x600@60032 (preferred)
00:00:01.353 [INFO] [backend/drm/drm.c:1376] Found connector 'VGA-1'
00:00:01.353 [INFO] [backend/drm/drm.c:1401] 'VGA-1' connected
00:00:01.353 [DEBUG] [backend/drm/drm.c:1207] Current CRTC: 37
00:00:01.353 [INFO] [util/env.c:9] Loading WLR_NO_HARDWARE_CURSORS option: 1
00:00:01.353 [DEBUG] [types/output/output.c:382] WLR_NO_HARDWARE_CURSORS set, forcing software cursors
00:00:01.353 [INFO] [backend/drm/drm.c:1217] Physical size: 470x300
00:00:01.353 [INFO] [backend/drm/drm.c:1274] Detected modes:
00:00:01.353 [INFO] [backend/drm/drm.c:1302]   1680x1050@59954 (preferred)
00:00:01.353 [INFO] [backend/drm/drm.c:1302]   1280x1024@75025 
00:00:01.353 [INFO] [backend/drm/drm.c:1302]   1280x1024@60020 
00:00:01.353 [INFO] [backend/drm/drm.c:1302]   1152x864@75000 
00:00:01.353 [INFO] [backend/drm/drm.c:1302]   1024x768@75029 
00:00:01.353 [INFO] [backend/drm/drm.c:1302]   1024x768@60004 
00:00:01.354 [INFO] [backend/drm/drm.c:1302]   800x600@75000 
00:00:01.354 [INFO] [backend/drm/drm.c:1302]   800x600@60317 
00:00:01.354 [INFO] [backend/drm/drm.c:1302]   640x480@75000 
00:00:01.354 [INFO] [backend/drm/drm.c:1302]   640x480@59940 
00:00:01.354 [INFO] [backend/drm/drm.c:1302]   720x400@70082 
00:00:01.354 [DEBUG] [backend/drm/drm.c:998] Reallocating CRTCs
00:00:01.354 [DEBUG] [backend/drm/drm.c:1009] State before reallocation:
00:00:01.354 [DEBUG] [backend/drm/drm.c:1023]   'LVDS-1' crtc=1 status=1 want_crtc=1
00:00:01.354 [DEBUG] [backend/drm/drm.c:1023]   'VGA-1' crtc=0 status=1 want_crtc=1
00:00:01.354 [DEBUG] [backend/drm/drm.c:1065] State after reallocation:
00:00:01.354 [DEBUG] [backend/drm/drm.c:1072]   'LVDS-1' crtc=1 status=1
00:00:01.354 [DEBUG] [backend/drm/drm.c:1072]   'VGA-1' crtc=0 status=1
00:00:01.354 [INFO] [backend/drm/drm.c:1434] connector LVDS-1: Requesting modeset
00:00:01.354 [DEBUG] [types/wlr_drm_lease_v1.c:531] Offering output LVDS-1
00:00:01.354 [DEBUG] [../src/output.c:98] enable output
00:00:01.354 [DEBUG] [../src/output.c:102] set preferred mode
00:00:01.354 [DEBUG] [types/output/render.c:224] Attaching empty buffer to output for modeset
00:00:01.354 [DEBUG] [types/output/render.c:75] Choosing primary buffer format XR24 (0x34325258) for output 'LVDS-1'
00:00:01.354 [DEBUG] [render/swapchain.c:105] Allocating new swapchain buffer
00:00:01.355 [DEBUG] [render/allocator/gbm.c:143] Allocated 1024x600 GBM buffer with format XR24 (0x34325258), modifier INVALID (0x00FFFFFFFFFFFFFF)
00:00:01.355 [DEBUG] [render/gles2/renderer.c:148] Created GL FBO for buffer 1024x600
00:00:01.360 [DEBUG] [GLES2] FS shader: 3 inst, 0 tex, 1 tex_indirect, 0 temps, 1 const
00:00:01.371 [DEBUG] [types/output/render.c:224] Attaching empty buffer to output for modeset
00:00:01.398 [INFO] [../src/output.c:179] enable adaptive sync on LVDS-1
00:00:01.398 [INFO] [../src/output.c:186] adaptive sync test passed
00:00:01.398 [DEBUG] [backend/drm/drm.c:520] connector LVDS-1: Failed to enable adaptive sync: connector doesn't support VRR
00:00:01.398 [ERROR] [../src/output.c:189] failed to commit output LVDS-1, destroying
00:00:01.399 [DEBUG] [types/output/cursor.c:432] Falling back to software cursor on output 'LVDS-1'
00:00:01.399 [INFO] [backend/drm/drm.c:1434] connector VGA-1: Requesting modeset
00:00:01.399 [DEBUG] [types/wlr_drm_lease_v1.c:531] Offering output VGA-1
00:00:01.399 [DEBUG] [../src/output.c:98] enable output
00:00:01.399 [DEBUG] [../src/output.c:102] set preferred mode
00:00:01.399 [DEBUG] [types/output/render.c:224] Attaching empty buffer to output for modeset
00:00:01.399 [DEBUG] [types/output/render.c:75] Choosing primary buffer format XR24 (0x34325258) for output 'VGA-1'
00:00:01.399 [DEBUG] [render/swapchain.c:105] Allocating new swapchain buffer
00:00:01.399 [DEBUG] [render/allocator/gbm.c:143] Allocated 1680x1050 GBM buffer with format XR24 (0x34325258), modifier INVALID (0x00FFFFFFFFFFFFFF)
00:00:01.400 [DEBUG] [render/gles2/renderer.c:148] Created GL FBO for buffer 1680x1050
00:00:01.429 [DEBUG] [types/output/render.c:224] Attaching empty buffer to output for modeset
00:00:01.494 [INFO] [../src/output.c:179] enable adaptive sync on VGA-1
00:00:01.494 [INFO] [../src/output.c:186] adaptive sync test passed
00:00:01.494 [DEBUG] [backend/drm/drm.c:520] connector VGA-1: Failed to enable adaptive sync: connector doesn't support VRR
00:00:01.494 [ERROR] [../src/output.c:189] failed to commit output VGA-1, destroying
00:00:01.495 [DEBUG] [types/output/cursor.c:432] Falling back to software cursor on output 'VGA-1'
00:00:01.496 [DEBUG] [types/output/cursor.c:432] Falling back to software cursor on output 'LVDS-1'
00:00:01.496 [DEBUG] [../src/server.c:466] WAYLAND_DISPLAY=wayland-0
00:00:01.496 [INFO] [../src/theme.c:336] read theme /usr/share/themes/Numix/openbox-3/themerc
00:00:01.823 [INFO] [../src/menu/menu.c:437] read menu file /home/benutzer/.config/labwc/menu.xml
00:00:02.287 [INFO] [../src/config/session.c:98] Updating dbus execution environment
00:00:02.329 [INFO] [../src/config/session.c:149] run autostart file /home/benutzer/.config/labwc/autostart
info: main.c:421: version: 1.13.1 +pgo +ime +graphemes -assertions
info: main.c:428: arch: Linux x86_64/64-bit
info: main.c:440: locale: de_DE.UTF-8
info: config.c:3003: loading configuration from /home/benutzer/.config/foot/foot.ini
info: fcft.c:338: fcft: 3.1.5 +graphemes -runs +svg(nanosvg) -assertions
info: fcft.c:377: fontconfig: 2.13.1, freetype: 2.12.1, harfbuzz: 5.2.0
2022-11-15 06:06:08 - [main.c:340] Found config * for output VGA-1 (Dell Inc. DELL E2210 [..])
2022-11-15 06:06:08 - [main.c:340] Found config * for output LVDS-1 (AU Optronics 0x60D2)
00:00:02.488 [DEBUG] [types/wlr_compositor.c:671] New wlr_surface 0x55e5660f7eb0 (res 0x55e5660c5c70)
00:00:02.488 [DEBUG] [types/wlr_layer_shell_v1.c:474] new layer_surface 0x55e5660c5d00 (res 0x55e5660c5e30)
info: fcft.c:838: ../subprojects/wlroots/types/scene/wlr_scene.c:69:3: runtime error: member access within misaligned address 0x000000000004 for type 'struct wlr_scene_tree', which requires 8 byte alignment
0x000000000004: note: pointer points here
<memory cannot be printed>
/usr/share/fonts/truetype/dejavu/DejaVuSansMono.ttf: size=8.00pt/8px, dpi=75.00
 err: wayland.c:1295: no compositor
 err: wayland.c:1729: failed to flush wayland socket: Connection reset by peer
info: main.c:680: goodbye
```
</details>
<details><summary>backtrace</summary>

```
#0  0x0000555555807ede in scene_node_init (node=0x55555648ca70, type=WLR_SCENE_NODE_TREE, parent=0x555500000000) at ../subprojects/wlroots/types/scene/wlr_scene.c:69
#1  0x0000555555808c98 in scene_tree_init (tree=0x55555648ca70, parent=0x555500000000) at ../subprojects/wlroots/types/scene/wlr_scene.c:142
#2  0x000055555580902e in wlr_scene_tree_create (parent=0x555500000000) at ../subprojects/wlroots/types/scene/wlr_scene.c:179
#3  0x000055555581b9d4 in wlr_scene_layer_surface_v1_create (parent=0x555500000000, layer_surface=0x55555648c7e0) at ../subprojects/wlroots/types/scene/layer_shell_v1.c:144
#4  0x0000555555703890 in new_layer_surface_notify (listener=0x7fffffffdd50, data=0x55555648c7e0) at ../src/layers.c:347
#5  0x00007ffff7f9b90c in wl_signal_emit_mutable () at /lib/x86_64-linux-gnu/libwayland-server.so.0
#6  0x0000555555879323 in layer_surface_role_commit (wlr_surface=0x555556499c00) at ../subprojects/wlroots/types/wlr_layer_shell_v1.c:367
#7  0x000055555584af17 in surface_commit_state (surface=0x555556499c00, next=0x555556499d88) at ../subprojects/wlroots/types/wlr_compositor.c:488
#8  0x000055555584b167 in surface_handle_commit (client=0x5555565d95e0, resource=0x555556465380) at ../subprojects/wlroots/types/wlr_compositor.c:504
#9  0x00007ffff74d8f7a in  () at /lib/x86_64-linux-gnu/libffi.so.8
#10 0x00007ffff74d840e in  () at /lib/x86_64-linux-gnu/libffi.so.8
#11 0x00007ffff74d8b0d in ffi_call () at /lib/x86_64-linux-gnu/libffi.so.8
#12 0x00007ffff7f9f981 in  () at /lib/x86_64-linux-gnu/libwayland-server.so.0
#13 0x00007ffff7f9abda in  () at /lib/x86_64-linux-gnu/libwayland-server.so.0
#14 0x00007ffff7f9d89a in wl_event_loop_dispatch () at /lib/x86_64-linux-gnu/libwayland-server.so.0
#15 0x00007ffff7f9b365 in wl_display_run () at /lib/x86_64-linux-gnu/libwayland-server.so.0
#16 0x0000555555704712 in main (argc=4, argv=0x7fffffffe478) at ../src/main.c:165
```

</details>

Reply:
Thanks. Very useful.
Looks like it's `wlr_output_commit_state()` that really tests if VRR is supported or not. It begs the question what `wlr_output_test_state()` actually achieves?
I've removed `output_destroy_notify()`, so think it should work now.

Reply:
The old i915 KMS drivers don't seem to support atomic commits, maybe that is the reason for the test being a success?